SPECS/python-sphinx/python-sphinx.spec

```diff
@@ -81,6 +84,7 @@ Requires:       python3-six
 Requires:       python3-snowballstemmer
 Requires:       python3-sphinx-theme-alabaster
 Requires:       python3-sphinxcontrib-websupport
+Provides:       python3-sphinx_rtd_theme = %{version}-%{release}
```

Comment:
I don't believe `python3-sphinx_rtd_theme` is provided by this package, are you sure this provide is right?

Reply:
You're right. Have removed the entry